### PR TITLE
🎉Source Twilio: New streams Roles and Services

### LINKED ISSUE #19744: Source Twilio: new stream roles
## Tell us about the problem you're trying to solve
The Role resource of Programmable Chat represents what a user can do within a Chat Service instance. Roles are scoped to either a [Service](https://www.twilio.com/docs/api/chat/rest/services) or a [Channel](https://www.twilio.com/docs/api/chat/rest/channels).

[Users](https://www.twilio.com/docs/chat/api/users) are assigned a role at the Service scope, which determines what they are can do within the Chat Service instance, such as create and destroy channels.

[Members](https://www.twilio.com/docs/chat/api/members) are assigned a role at the Channel scope. This determines what they are able to do within a particular channel, such as invite users to be members of the channel, post messages, and remove members from the channel.
## Describe the solution you’d like
Research rpos and cons for implenting this stream

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #19744

## Enhancement
New streams: 

- Services
- Roles

## Recommended reading order
1. `source_twilio/source.py`
2. `source_twilio/streams.py`
3.  `source_twilio/schemas/roles.json`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by sh4sh** [comment]
Note: before merging, we should bump the version in the connector's metadata.yaml as part of the new connector image publishing process

File: docs/integrations/sources/twilio.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ The Twilio source connector supports the following [sync modes](https://docs.air
 * [Outgoing Caller Ids](https://www.twilio.com/docs/voice/api/outgoing-caller-ids#outgoingcallerids-list-resource)
 * [Queues](https://www.twilio.com/docs/voice/api/queue-resource#read-multiple-queue-resources)
 * [Recordings](https://www.twilio.com/docs/voice/api/recording#read-multiple-recording-resources) \(Incremental\)
+* [Services](https://www.twilio.com/docs/chat/rest/service-resource#create-a-service-resource)
```
Comment (sh4sh) [inline docs/integrations/sources/twilio.md:78]:
```suggestion
* [Services](https://www.twilio.com/docs/chat/rest/service-resource#read-multiple-service-resources)
```

File: airbyte-integrations/connectors/source-twilio/source_twilio/streams.py [hunk 2]
```diff
@@ -443,6 +444,32 @@ class Recordings(IncrementalTwilioStream, TwilioNestedStream):
     upper_boundary_filter_field = "DateCreated<"
     cursor_field = "date_created"
 
+class Services(TwilioStream):
+    """
+    https://www.twilio.com/docs/chat/rest/service-resource#create-a-service-resource
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-twilio/source_twilio/streams.py:449]: This link directs me to documentation for a `POST` request, I believe you meant this one:

```suggestion
    https://www.twilio.com/docs/chat/rest/service-resource#read-multiple-service-resources
```

**Comment by jrolom** [comment]
Hi @sh4sh ! Where is this PR stuck?

**Review by marcosmarxm — APPROVED** [review]

File: airbyte-integrations/connectors/source-twilio/source_twilio/streams.py [hunk 3]
```diff
@@ -19,6 +19,7 @@
 from pendulum.datetime import DateTime
 from requests.auth import AuthBase
 
+TWILIO_CHAT_BASE = "https://chat.twilio.com/v2/"
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-twilio/source_twilio/streams.py:22]: Need add to `allowedHosts` in metabese file

**Review by lazebnyi — APPROVED** [review]
LGTM! Minor changes required.

**Pull request merged**